### ISSUE TITLE
Remove <p> markup inside of nav links

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#244]
Reference: [WCAG 4.1.2 - Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)

### The Problem
Within the `<nav>` markup, within each nav link (`<a>`), there is a `<p class="navOption">`. Don't think there should be a paragraph within a nav link. 

### Suggestion
Give the `<a>` element a class of `.navOption` and remove the `<p>`. 

### Acceptance Criteria
No `<p>`s exist inside `<nav>`.

### Definition of Done
- [ ] All tests (unit/functional/security/code cov.) pass
- [ ] Code review has been done
- [ ] Pipeline builds
- [ ] PO signs off on the build

**Comment by thegentlemanphysicist** [comment]
This was fixed with the refactor I did of #240.

**Comment by thegentlemanphysicist** [comment]
https://digital-gov-frontend-pr-348-xdwiwd-dev.pathfinder.gov.bc.ca/
